docs/resources/Guild_Scheduled_Event.md

```diff
@@ -0,0 +1,134 @@
+| scheduled_start_time | ISO8601 timestamp                                                                                                           | the time to schedule the scheduled event                                        |
+| scheduled_end_time   | ISO8601 timestamp                                                                                                           | the time when the scheduled event is scheduled to end                           |
+| description          | string                                                                                                                      | the description of the scheduled event                                          |
+| entity_type          | [event entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types)  | the entity type of the scheduled event                                          |
+| status               | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)             | the status of the scheduled event                                               |
+
+## Delete Guild Scheduled Event % DELETE /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events/{guild_scheduled_event.id#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object}
+
+Delete a guild scheduled event. Returns a `204` on success.
+
+## Get Guild Scheduled Event Users % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events/{guild_scheduled_event.id#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object}/users
+
+Get a list of users RSVP'd to the guild scheduled event. Returns a list of [user](#DOCS_RESOURCES_USER/user-object) objects on success with an optional `guild_member` property for each user if `with_member` query parameter is passed in.
+
+###### Query String Params
+
+| Field        | Type    | Description                                                                    | Default |
+| ------------ | ------- | ------------------------------------------------------------------------------ | ------- |
+| limit?       | number  | how many users to receive from the event                                       | 100     |
+| with_member? | boolean | include guild member data. attaches `guild_member` property to the user object | false   |
+
+###### Response Structure
+
+| Field         | Type                                                                                                            |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| users         | array of [user](#DOCS_RESOURCES_USER/user-object) objects with an optional `guild_member` property for each user|
```

Comment:
Line 134 has a missing space for the end of `user` which appears like it can generate documentation issues, looking at the other examples. I suggest adding a space.